worth2/templates/main/journal_base.html

```diff
@@ -1,6 +1,6 @@
 {% extends 'base.html' %}
 
-{% block title %}Journal for Session {{ session_num }}{% endblock %}
+{% block title %}Journal for Session {{ session_num }} | Participant: {{ participant.study_id }}{% endblock %}
```

Comment:
right - by internal I meant to the study, not the system. If they're not worried about exposing it, that's fine then.